Justfile

```diff
@@ -162,6 +164,13 @@ build-frontend: pnpm-install
 build-extension: pnpm-install
   @mk {{EXTENSION_OUT}} : frontend/vscode/src frontend/vscode/package.json -- "cd frontend/vscode && rm -f ftl-*.vsix && pnpm run compile"
 
+# Build the sqlc-ftl-gen plugin, used to generate FTL schema from SQL
+build-sqlc-gen-ftl: build-protos update-sqlc-plugin-codegen-proto rust-toolchain-install
```

Comment:
I don't think we need to download the proto on every build?

Reply:
true 😬